### PR TITLE
CASH-684 Add featured hero loan thermometer experiment

### PR DESCRIPTION
**Pull request by foochris**

### CODE REVIEW

File: src/pages/Lend/LendByCategoryPage.vue [hunk 1]
```diff
@@ -323,6 +323,8 @@ export default {
 					client.query({ query: experimentQuery, variables: { id: 'image_enhancement' } }),
 					// experiment: featured hero loan
 					client.query({ query: experimentQuery, variables: { id: 'featured_hero_loan_v2' } }),
+					// experiment: featured hero loan thermometer
```
Comment (mcstover) [inline src/pages/Lend/LendByCategoryPage.vue]: this should replace featured_hero_loan_v2 right?

File: src/components/LoansByCategory/FeaturedHeroLoan.vue [hunk 2]
```diff
@@ -194,7 +194,7 @@ export default {
 			return !!(this.loanUse.length > this.loanUseMaxLength);
 		},
 		showFundraisingThermometer() {
-			return _get(this.experimentData, 'featured_hero_loan_fundraising_thermometer') === 'variant-c';
+			return _get(this.experimentData, 'featured_hero_loan_fundraising_thermometer') === 'variant-b';
```
Comment (mcstover) [inline src/components/LoansByCategory/FeaturedHeroLoan.vue]: I don't think this matches you code below which has variant-c showing the thermo

File: src/pages/Lend/LendByCategoryPage.vue [hunk 3]
```diff
@@ -398,20 +400,31 @@ export default {
 		});
 
 		// eslint-disable-next-line max-len
-		this.featuredHeroLoanExperimentVersion = _get(featuredHeroLoanExperimentVersionArray, 'experiment.version') || null;
+		const featuredHeroLoanExperimentVersion = _get(featuredHeroLoanExperimentVersionArray, 'experiment.version') || null;
 
-		if (this.featuredHeroLoanExperimentVersion === 'variant-a') {
+		console.log(featuredHeroLoanExperimentVersion);
```
Comment (mcstover) [inline src/pages/Lend/LendByCategoryPage.vue]: remove

**Comment by foochris** [comment]
@mcstover PR updated. DEV and PROD updated with new experiment values (`uiexp.featured_hero_loan_v3`).

**Comment by foochris** [comment]
@mcstover Experiment setting added to QA and enabled to `true` on DEV and QA, but not PROD.

**Pull request merged**